geoserve/geoserve.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 const (
-	DB_URL = "https://geolite.maxmind.com/download/geoip/database/GeoLite2-City.mmdb.gz"
+	DB_URL = "https://s3.amazonaws.com/lantern/GeoLite2-City.mmdb.gz"
```

Comment:
This is the only real change to fix the app crashing.

Reply:
I'm not sure if it's a violation of MaxMind's term of use by making it publicly accessible. Maybe better protect it with a token?

Reply:
Good idea. Another option, if we're having to manage the file manually, is to just embed it with our binary and deploy whenever we update the file.

Reply:
Also, if we find a way to keep it private, we might as well switch to the paid version which gives improved accuracy.

Reply:
Oh well the file is still available to download via code, see https://dev.maxmind.com/geoip/geoipupdate/#Direct_Downloads. Will do as we surely need to keep the file up-to-date.